### ISSUE TITLE
move config to `dfhack-config`

### ISSUE THREAD

**Issue by ab9rf** [DFHack/stonesense#105]
so it doesn't get overwritten with dfhack updates

[discord discussion](https://discord.com/channels/793331351645323264/807444467140788254/1239747641416482826)

**Comment by realSquidCoder** [comment]
Thinking about it, doesn't this issue duplicate the problem of [#90](https://github.com/DFHack/stonesense/issues/90)?

**Comment by ab9rf** [comment]
> Thinking about it, doesn't this issue duplicate the problem of [#90](https://github.com/DFHack/stonesense/issues/90)?

no, it's not a duplicate, although it's likely that a good solution will resolve both of these issues

**Comment by realSquidCoder** [comment]
Ok, agreed

**Comment by realSquidCoder** [comment]
afaik this is now handed in #120
